test/library/calibration/test_half_angle.py

```diff
@@ -0,0 +1,57 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test rough amplitude calibration experiment classes."""
+
+from test.base import QiskitExperimentsTestCase
+
+from qiskit import pulse
+from qiskit.circuit import Parameter
+
+from qiskit_experiments.exceptions import CalibrationError
+from qiskit_experiments.calibration_management.basis_gate_library import FixedFrequencyTransmon
+from qiskit_experiments.calibration_management import Calibrations
+from qiskit_experiments.library import HalfAngleCal
+from qiskit_experiments.test.pulse_backend import SingleTransmonTestBackend
+
+
+class TestHalfAngleCal(QiskitExperimentsTestCase):
```

Comment:
Out of scope for this PR, but it would be good to have a basic test of the class working.

Reply:
I agree, I already opened an issue - #1123 .